dime-coding-standards/checklists/tex/code-review-package-checklist

```diff
@@ -0,0 +1,133 @@
+\rfoot{\includegraphics[height=1cm,keepaspectratio]{../../img/analytics}}
+
+% Put checkbox to the left of the text
+\def\LayoutCheckField#1#2{% label, field
+	#2 #1%
+}
+
+% Line spacing
+\onehalfspacing
+
+% Define DIME Analytics visual identity colors
+\definecolor{fontcolor}{HTML}{7A0569}
+
+\titleformat{\section}%
+{\Large\rmfamily\bf\color{fontcolor}}% format applied to label+text
+{\llap{\colorbox{fontcolor}{\parbox{1.5cm}{\hfill\huge\color{fontcolor}\thesection}}}}% label
+{2pt}% horizontal separation between label and title body
+{}% before the title body
+[]% after the title body
+
+\titleformat{\subsection}%
+{\large\rmfamily\color{fontcolor}}% format applied to label+text
+{}% label
+{1.5pt}% horizontal separation between label and title body
+{}% before the title body
+[]% after the title body
+
+\newcommand{\dimeCheckBox}[1]{\CheckBox[height=0.01cm, width=0.4cm, bordercolor=gray]{#1}}
+\newcommand{\dimeTextField}[3]{\TextField[name=#1, height=0.3cm, width=#2, bordercolor=gray]{#3}}
+
+
+\newcommand{\titleBox}[1]{
+	\begin{tcolorbox}
+		[colframe = fontcolor,
+		colback = fontcolor,
+		sharp corners,
+		halign = flush center,
+		valign = center,
+		height = 0.3\textwidth,
+		after skip = 1cm]
+		#1
+	\end{tcolorbox}
+}
+
+\begin{document}
+	\begin{fullwidth}
+		
+		\titleBox{
+			\textcolor{white}{\LARGE{\textbf{DIME Analytics \\ Code Review Package Checklist}} \\
+				\Large\textbf{{v2.0}}}
+		}
+		
+		\section*{Required Content and Practices}
+		
+		The code review package must be shared as a .zip file, GitHub link or shared folder containing the items listed below.
+		All file paths in the code \textbf{must} use only forward slashes, i.e. \texttt{C:/users/...}
+		
+		\subsection{1. \, Package components}
+		
+		\begin{itemize}
+			\setlength\itemsep{-0.1em}
+			\item[] \dimeCheckBox{All scripts to be reviewed.}
+			\item[] \dimeCheckBox{All data necessary to run the code to be reviewed - in deidentified form.}
+			\item[] \dimeCheckBox{A README. file that contains the following information:}
+                        \begin{itemize}
+                            \item[] \dimeCheckBox{Purpose of each folder}
+                            \item[] \dimeCheckBox{Purpose of each script (if more than one script is used)}
+                            \item[] \dimeCheckBox{Is all confidential data de-identified? (if provided in the review)} 
+                             \item[] \dimeCheckBox{Outputs produced (if part of the review)}
+                             \item[] \dimeCheckBox{Which script creates which output}
+                             
+                            
+                        \end{itemize}
+			\item[] \dimeTextField{master}{11cm}{README. file path and name:}
+			\item[] \dimeCheckBox{A \textbf{master script} that runs all the necessary code and allows the reviewer to adjust file paths only once.}
+		\end{itemize}
+		
+		\vspace{.6em}
+		
+		\subsection{2. \, Master script requirements}	
```

Comment:
the overall structure needs some reorganization. Point 1 is package components, but 2, 3, and 4 seem like they are more details on some parts of the components. would it make sense to integrate them? for example, under , Package Components there's a checkbox for "All data necessary to run the code to be reviewed - in deidentified form.", but under Data there is also a checkbox that says "All data necessary to run the code to be reviewed is included". this seems redundant.